### PR TITLE
docs: Template prometheus values based on Karpenter namespace

### PR DESCRIPTION
**Pull request by RobertNorthard**
Fixes #N/A

**Description**
The Monitoring with Grafana part of the getting started guide for Karpenter hardcodes the Karpenter namespace to "kube-system" in the prometheus scrape targets configuration.  This is a configurable option in the walk-through. This PR templates the prometheus values file.

**How was this change tested?**

**Does this change impact docs?**
- [x] Yes, PR includes docs updates 
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: website/content/en/docs/getting-started/getting-started-with-karpenter/prometheus-values.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ extraScrapeConfigs: |
       - role: endpoints
         namespaces:
           names:
-          - kube-system
+          - $KARPENTER_NAMESPACE
```
Comment (engedaam) [inline website/content/en/docs/getting-started/getting-started-with-karpenter/prometheus-values.yaml:16]: Thank you for opening this PR! Would you be willing add this changes to v0.32 and v0.33?

Comment (RobertNorthard) [reply website/content/en/docs/getting-started/getting-started-with-karpenter/prometheus-values.yaml:16]: Thank you, I've updated 0.32 and 0.33 as well now.

File: website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step09-add-prometheus-grafana.sh [hunk 2]
```diff
@@ -3,8 +3,9 @@ helm repo add prometheus-community https://prometheus-community.github.io/helm-c
 helm repo update
 
 kubectl create namespace monitoring
+export KARPENTER_NAMESPACE=${KARPENTER_NAMESPACE:-kube-system}
```
Comment (engedaam) [inline website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step09-add-prometheus-grafana.sh:6]: Everything else looks good, just one nit. We already export the `KARPENTER_NAMESPACE` in the beginning of the getting started guide. Is it necessary to export it again here? https://karpenter.sh/docs/getting-started/getting-started-with-karpenter/#2-set-environment-variables

Comment (RobertNorthard) [reply website/content/en/docs/getting-started/getting-started-with-karpenter/scripts/step09-add-prometheus-grafana.sh]: Thanks - I decided to remove the export as we are not doing the same for the KARPENTER_VERSION. My assumption was if that readers do not follow the tutorial in one go (and revisit this optional part later) readers might forget to export the environment variables.

**Review by engedaam — APPROVED** [review]
LGTM 🚀

**Pull request merged**